client/post-editor/editor-revisions/dialog.jsx

```diff
@@ -93,6 +98,7 @@ class PostRevisionsDialog extends PureComponent {
 				isVisible={ isVisible }
 				onClose={ closeDialog }
 			>
+				<CloseOnEscape onEscape={ this.onEscape } />
```

Comment:
Is there any reason we can't just do this here?
`<CloseOnEscape onEscape={ closeDialog } />`